docs/source/getting-started.rst

```diff
@@ -94,7 +94,93 @@ or for ``Ninja``:
 
 .. note::
 
-        Loading the bitstream into an FPGA can be done outside of the Symbiflow with tools that support the target architecture, such as for example `Vivado` or `xc3sprog` for Xilinx 7-Series FPGAs or ``iceprog`` for Lattice iCE40.
+        Loading the bitstream into an FPGA can be done outside of the Symbiflow with tools that support the target architecture. A good example is `openFPGALoader` or others such as `Vivado` and `xc3sprog` for Xilinx 7-Series FPGAs or ``iceprog`` for Lattice iCE40.
+
+OpenFPGALoader
+++++++++++++++
+
+OpenFPGALoader is an universal utility for programming the FPGA devices that is
+a great alternative to OpenOCD. It supports many different boards with FPGAs
+based on the architectures including xc7, ECP5, iCE40 and many more. It can utilize
+a variety of the programming adapters based on JTAG, DAP interface, ORBTrace,
+DFU and FTDI chips.
+
+Installing OpenFPGALoader
+*************************
+
+The most reliable way of getting OpenFPGALoader installed on your computer is to
```

Comment:
I don't think the most reliable way is building a tool from sources. openFPGALoader is available in several packaging solutions (such as containers or MSYS2, see https://github.com/hdl/packages; also https://github.com/hdl/conda-prog/pull/22), and CI generates "nightly" packages (https://github.com/trabucayre/openFPGALoader/releases/tag/nightly). Therefore, I believe the docs in symbiflow should recommend using one of those pre-built solutions, if suitable. 

Moreover, instructions about how to build and install openFPGALoader are better contributed in the openFPGALoader repo (https://github.com/trabucayre/openFPGALoader/blob/master/INSTALL.md), since that knowledge is not specific about SymbiFlow. It's ok to explain if some specific setting or variable needs to be used in the context of SymbiFLow.

Reply:
https://github.com/trabucayre/openFPGALoader/pull/102